recipes/gravelamps/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "gravelamps" %}
+{% set version = "2.3.1rc14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/mick.wright/Gravelamps/-/archive/{{ version }}/Gravelamps-{{ version }}.tar.gz
+  sha256:  95569b7a1bf3b44574b3915da1bd7404434d8c3823b8d7e62849eab8d4950bf5
+
+build:
+  entry_points:
+    - gravelamps_inference=gravelamps.inference:main
+    - gravelamps_generate_lens=gravelamps.generate_lens:main
+    - gravelamps_generate_interpolator_data=gravelamps.lensing.generic:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [not linux or py<=39]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - sysroot_linux-64
```

Comment:
I've removed the package.

I had been having issues with the package not having the right version of glibc which was actually a build order issue rather than an actual need for an updated glibc. It's been resolved, so neither the package nor an updated glibc is needed, and 2.17 works fine.

Reply:
For clarity, the default version of glibc in the build environment is older than 2.17, and the newest we provide is 2.17.

Reply:
Ah, then I should have said the default worked fine.